storage/caching/caching_test.go

```diff
@@ -68,7 +70,7 @@ func TestReadTypeDefinition(t *testing.T) {
 				continue
 			}
 
-			cacheKey := strings.Join([]string{test.store, modelID, test.name}, Separator)
+			cacheKey := strings.Join([]string{test.store, test.model.Id, test.name}, Separator)
```

Comment:
Can we join this with `fmt.Sprintf(...)` instead? It's less obvious the resulting string using `strings.Join(..)`.

Reply:
I can do this, but it is quite a bit worse honestly:
```
cacheKey := fmt.Sprintf("%s%s%s%s%s", test.store, separator, test.model.Id, separator, test.name)
```
You still like it?

Reply:
Oh, sorry I don't think I was clear. Variablizing the separator and using strings.Join just adds indirection and makes it hard to understand the resulting key.

I was proposing `fmt.Sprintf("%s:%s:%s", test.store, test.model.Id, test.name)`, which is better IMO to understand.

Reply:
Updated to remove the separator indirection as discussed offline:
```
cacheKey := fmt.Sprintf("%s:%s:%s", test.store, test.model.Id, test.name)
```